docs/getting-started/startup-basics.md

```diff
@@ -0,0 +1,61 @@
+# Startup Basics
+
+This guide was last updated for v0.8.2
+
+---
+
+This guide applies to command-line application users. Paymetheus users can safely ignore the use of config files - Paymetheus handles configuration automatically. It is also worth noting that some of our guides show configuration file settings and other guides show startup command flags. 
+
+---
+
+## Configuration Files
+
+All of the Decred software, when started, reads from a configuration file to determine which settings it should enable/disable/set during that initial load. All of the command line startup flags `(e.g. dcrwallet --testnet)` can be replaced by settings within the appropriate configuration file `(e.g. dcrwallet --testnet could be replaced by testnet=1 in dcrwallet.conf)`.
+
+These configuration files are located with the application home directory of the application. The location of these default home directories for Windows, macOS/OSX, and Linux are listed below:
+
+Windows:
+
+    C:\Users\<username>\AppData\Local\Dcrwallet\
+    C:\Users\<username>\AppData\Local\Dcrd\
+    C:\Users\<username>\AppData\Local\Dcrctl\ 
+
+macOS: 
+
+    ~/Library/Application Support/Dcrwallet/
+    ~/Library/Application Support/Dcrd/
+    ~/Library/Application Support/Dcrctl/
+    
+Linux: 
+    
+    ~/.dcrwallet/
+    ~/.dcrd/
+    ~/.dcrctl/
+
+Each of these folders is allowed it's own `.conf` file, named after the individual application (`e.g. Dcrd uses dcrd.conf`). Please also note that `Dcrd` and `Dcrwallet` home directories are automatically created when each application is first launched. You will have to manually create a `Dcrctl` home directory to utilize a config file.
```

Comment:
Do we normally capitalize the names of dcrd, dcrwallet, and dcrctl?  I think not.